src/reducers/index.js

```diff
@@ -57,6 +57,24 @@ export const removeResource = (state, action) => {
   return validate(newState)
 }
 
+// TODO: could combine setPublishError() and setRetrieveError() with arg for single word diff in error message
```

Comment:
Could you create a ticket for this TODO and move some of detail from this comment to the ticket?

Reply:
I'm vastly shortening the comment - is that your main objection to it?

I think making an issue is premature, but will create one if you think it's worthy.